bs/__tests__/tablecloth_test.ml

```diff
@@ -3,6 +3,13 @@ open Jest
 open Expect
 
 let () =
+  describe "Result" (fun () ->
+    describe "fromOption" (fun () ->
+      test "maps None into Error" (fun () -> expect Result.(fromOption ~error:"error message" None) |> toEqual (Belt.Result.Error "error message"));
+      test "maps Some into Ok" (fun () -> expect Result.(fromOption ~error:"error message" (Some 10)) |> toEqual (Belt.Result.Ok 10));
```

Comment:
Did you see my suggestions on the previous commit?

Reply:
@Dean177 whoops I forgot to update those. What do you want me to do? I can open a PR dropping the `Belt.Result.` namespace.